libraries/ESP32/examples/Template/ExampleTemplate/README.md

```diff
@@ -98,21 +98,21 @@ Chip ID: 3957392
 * **Programming Fail:** If the programming/flash procedure fails, try reducing the serial connection speed.
 * **COM port not detected:** Check the USB cable and the USB to Serial driver installation.
 
-If the error persist, you can ask for help at the official [ESP32 forum](https://esp32.com) or see [Contribute](#contribute).
+If the error persists, you can ask for help at the official [ESP32 forum](https://esp32.com) or see [Contribute](#contribute).
 
 ## Contribute ==(REQUIRED)==
 
-==*Do not change! Keep as is.*==
+==*Do not change! Keep it as is.*==
 
 To know how to contribute to this project, see [How to contribute.](https://github.com/espressif/arduino-esp32/blob/master/CONTRIBUTING.rst)
 
 If you have any **feedback** or **issue** to report on this example/library, please open an issue or fix it by creating a new PR. Contributions are more than welcome!
 
-Before creating a new issue, be sure to try the Troubleshooting and to check if the same issue was already created by someone else.
+Before creating a new issue, be sure to try Troubleshooting and check if the same issue was already created by someone else.
 
 ## Resources ==(REQUIRED)==
 
-==*Do not change here! Keep as is or add only relevant documents/info for this example. Do not add any purchase link/marketing stuff*==
+==*Do not change here! Keep it as is or add only relevant documents/info for this example. Do not add any purchase link/marketing stuff*==
 
 * Official ESP32 Forum: [Link](https://esp32.com)
 * Arduino-ESP32 Official Repository: [espressif/arduino-esp32](https://github.com/espressif/arduino-esp32)
```

Comment:
@pedrominatel As this part was not edited and hidden in the PR, we should add ESP32-S3 datasheet link too. To have it in all future example READMEs.